### ISSUE TITLE
Dropdown | Accessibility update & add missing variables

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng-sass-theme#61]
Add variables:

- inputListItemFocusBg
- inputListItemTextFocusColor
- highlightFocusBg
